Sources/SwiftFrame/Render.swift

```diff
@@ -4,17 +4,51 @@ import SwiftFrameCore
 
 struct Render: ParsableCommand {
 
-    @Argument(help: "Read configuration values from the specified file", completion: .list(["config", "json", "yml", "yaml"]))
+    // MARK: - Arguments
+
+    @Argument(
+        help: "Read configuration values from the specified file",
+        completion: .list(["config", "json", "yml", "yaml"])
+    )
     var configFilePath: String
 
-    @Flag(name: .shortAndLong, help: "Prints additional information and lets you verify the config file before rendering")
+    // MARK: - Opt-In Features
+
+    @Flag(
+        name: .shortAndLong,
+        help: "Prints additional information and lets you verify the config file before rendering"
+    )
     var verbose = false
 
-    @Flag(name: .long)
-    var noManualValidation = false
+    @Flag(
+        name: .long,
+        help: "Pauses after parsing the config file to let you verify the contents"
+    )
+    var manualValidation = false
+
+    @Flag(
+        name: .long,
+        help: "Outputs the whole image canvas into the output directories before slicing it up into the correct screenshot sizes. Helpful for troubleshooting"
+    )
+    var outputWholeImage = false
+
+    // MARK: - Opt-Out Features
+
+    @Flag(
+        name: .long,
+        inversion: .prefixedNo,
+        help: "Controls whether logged output will be colored or now"
+    )
+    var colorOutput = true
+
+    @Flag(
+        name: .long,
+        inversion: .prefixedNo,
```

Comment:
Hmm I see now that this seems to be a common convention. At least based on the `inversion` option